### PR TITLE
Updating text for a second major revision

### PR DESCRIPTION
**Pull request by matthewturk**
I've updated much of the text in this, and I plan to start using it as part of an onboarding checklist to make sure expectations are covered.  Suggestions are welcome.

### CODE REVIEW

File: source/expectations.rst [hunk 1]
```diff
@@ -78,43 +87,31 @@ other projects as they become available.
...
+We utilize `git-annex` for internal data sharing, and for sharing data
+externally, we recommend utilizing either `hub.yt <https://hub.yt>`_ or another
+service such as the `Illinois Data Bank <https://databank.illinois.edu>`_.
+
+The canonical `git-annex` repository is at
+https://github.com/data-exp-lab/datasets , and it utilizes the Box service at
+University of Illinois for backend storage.
+
+In that repository, the `README.md` file describes how to utilize `git-annex`
+for sharing data internally, as well as how to add a new dataset.
```
Comment (ngoldbaum) [inline source/expectations.rst:99]: this is news to me

**Review by ngoldbaum — APPROVED** [review]
LGTM other than the note about git annex which I was previously unaware of

File: source/checklist.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ 1. Go over this document with Matt
+    a. Code of Conduct
+    b. Communication modes
+    c. Expectations around openness
+ 2. Get added to the DXL groups
+    a. Mailinglist
+    b. Github (and Bitbucket)
+    c. Slack
+    d. Box folder for `git-annex`
+ 3. Add yourself to the `data-exp-lab.github.io` webpage (and a blog post about
```
Comment (munkm) [inline source/checklist.rst:15]: You use "blog post" here but then on line 9 of `communication.rst` you use "news items". I would maybe change this to "blog post" so it's clear they're not expected to write something long for their welcome-to-the-lab-type post.

File: source/communication.rst [hunk 3]
```diff
@@ -1,6 +1,17 @@
 Communication
 =============
 
+Group Web Site
+--------------
+
+We have a web site, stored in a repository at
+https://github.com/data-exp-lab/data-exp-lab.github.io/ .  You are expected to
+post news articles when you do things of note, as well as to update project
+information on that page.  All updates are conducted via pull request and git
```
Comment (munkm) [inline source/communication.rst]: Do you want the website to always be updated with PRs? I've been committing directly to it with news items.

**Comment by matthewturk** [comment]
I've talked about it a few times in group meeting and on slack

File: source/expectations.rst [hunk 4]
```diff
@@ -78,43 +87,31 @@ other projects as they become available.
...
-  /dpool/mturk/HRRR-20140726/dir2/*
-
-and so on.  In the near-term future, we will be deploying a data cataloging
-system which will be fed with the data from README files, and which will be the
-primary cataloging system in the future.
-
-Wherever possible, names should conform to including either a date (if the data
-is tied to a specific time, parameter query or observation) or a
-semantically-meaningful name (if it is part of a larger project.)
+We utilize `git-annex` for internal data sharing, and for sharing data
```
Comment (munkm) [inline source/expectations.rst:90]: Here you're talking about using git annex for data sharing and you've removed some of the documentation about storing data in /dpool/. In `infrastructure.rst` you mention dpool as a computing resource for file storage. Maybe add in information about how you expect these to interoperate?

File: source/infrastructure.rst [hunk 5]
```diff
@@ -39,23 +37,40 @@ NCSA has an allocation on the Illinois Campus Cluster
...
+We have a few things that you can use varying in degrees of playfulness and
+utility.
+
+ * HTC Vive
+ * Two Oculus Rift consumer products, one development kit (with Oculus
+   Touch controllers)
+ * `Project Tango tablet <https://en.wikipedia.org/wiki/Tango_(platform)>`_
+ * Chromecast
+ * Dell android tablet
+ * `Dark Side lego kit
```
Comment (munkm) [inline source/infrastructure.rst:57]: <3 the lego kit

**Review by munkm — APPROVED** [review]
Looks good to me!

File: source/expectations.rst (hunk 4 above)
Comment (matthewturk) [reply source/expectations.rst:90]: Whoops, gotta get rid of the dpool stuff.

**Pull request merged**